3-PB/Documentazione_esterna/Verbali/Verbale_2024_02_28/info.tex

```diff
@@ -0,0 +1,28 @@
+\section{Informazioni della riunione} \label{sec:info}
+La riunione è stata svolta da remoto, tramite piattaforma Google Meet, dalle ore 16:00 alle ore 17:00, in presenza di Carlo Davanzo ed Emilio Bendotti, rappresentanti dell’azienda AzzurroDigitale.
+
+\subsection{Lista partecipanti} \label{subsec:partecipanti}
+{
+    \setlength{\tabcolsep}{10pt}
+    \renewcommand{\arraystretch}{1.5}
+    \rowcolors{2}{oddrow}{evenrow}
+    \begin{tabular}{| l | c |}
+        \hline
+        \rowcolor{headerrow}\textbf{\textcolor{white}{Partecipante}}& \textbf{\textcolor{white}{Presenza}} \\
+        \hline
+        Andrea Cecchin & \textcolor{cmarkcolor}{\ding{51}}\\
+        \hline
+        Marco Dolzan & \textcolor{xmarkcolor}{\ding{55}}\\
+        \hline
+        Francesco Ferraioli & \textcolor{cmarkcolor}{\ding{51}}\\
+        \hline
+        Francesco Giacomuzzo & \textcolor{cmarkcolor}{\ding{51}}\\
+        \hline
+        Leonardo Lago & \textcolor{cmarkcolor}{\ding{51}}\\
+        \hline
+        Giovanni Menon & \textcolor{cmarkcolor}{\ding{51}}\\
+        \hline
+        Anna Nordio & \textcolor{cmarkcolor}{\ding{51}}\\
+        \hline
+    \end{tabular}
+}
```

Comment:
```suggestion
}
\\ \\In rappresentanza di AzzurroDigitale:\\ \\
{
    \setlength{\tabcolsep}{10pt}
    \renewcommand{\arraystretch}{1.5}
    \rowcolors{2}{oddrow}{evenrow}
    \begin{tabular}{|l|}
        \hline
        \rowcolor{headerrow}\textbf{\textcolor{white}{Partecipante}} \\
        \hline
        Carlo Davanzo\\
        \hline
        Emilio Bendotti\\
        \hline
    \end{tabular}
}
```

In generale, ti sconsoglio di usare il template dei verbali come copia su cui lavorare, considerando non sono mai aggiornate. Ti consiglio di prendere l'ultimo verbale di categoria scritto.